PWGDQ/Tasks/filterPP.cxx

```diff
@@ -142,7 +159,7 @@ struct DQBarrelTrackSelectionTask {
 
   float* fValues; // array to be used by the VarManager
 
-  Configurable<std::string> fConfigCuts{"cfgBarrelTrackCuts", "jpsiPID1", "Comma separated list of barrel track cuts"};
+  Configurable<std::string> fConfigCuts{"cfgBarrelTrackCuts", "jpsiKineAndQuality,jpsiPID1", "Comma separated list of barrel track cuts"};
```

Comment:
I suggest we keep just the jpsiPID1 cut as default. The jpsiKineAndQuality generates too much combinatorics